### ISSUE TITLE
SystemConfig getListProperty reads empty declared variable as List with one element

### ISSUE THREAD

**Issue by garyluoex** [yahoo/fili#101]
* Might be an ` org.apache.commons.configuration.CompositeConfiguration` bug
* Here is [`getListProperty`](https://github.com/yahoo/fili/blob/369787a86097e19c672072c8fe4e5b76e4fc012a/fili-system-config/src/main/java/com/yahoo/bard/webservice/config/SystemConfig.java#L141)

**Comment by cdeszaq** [comment]
Good catch! We'll try to get this fixed before the next stable release (ie. before we call v0.7.x stable)

**Comment by cdeszaq** [comment]
When we go to fix this, it _may_ be worthwhile to tackle upgrading to the v2 line of the Apache Configuration. Based on the [upgrade guide](https://commons.apache.org/proper/commons-configuration/userguide/upgradeto2_0.html), it looks like list handling is one area that has gotten a fair bit of love and attention, as well as hierarchical configurations, which is what we've got going on.
